packages/sqflite/example/lib/raw_test_page.dart

```diff
@@ -113,7 +113,7 @@ class RawTestPage extends TestPage {
         await db
             .execute('CREATE TABLE Test (id INTEGER PRIMARY KEY, name TEXT)');
 
-        Future _test(int i) async {
+        Future onTransaction(int i) async {
```

Comment:
nit: Might be renamed to `testItem`.

https://github.com/tekartik/sqflite/commit/75ad6b920ae6b398264a6ac87a0b1cafd9f73797#diff-aee91260b322128a1d796ce755e33bd186c397f7404bbc2af41b9728b45de9c7

Reply:
Done.